expense_tracker/lib/expense_tracker_app.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:expense_tracker/expenses_screen/expenses_screen.dart';
+import 'package:flutter/material.dart';
+
+/// color scheme for the app
+final kColorScheme = ColorScheme.fromSeed(
+  seedColor: const Color.fromARGB(255, 95, 60, 180),
+);
+
+/// dark color scheme for the app
+final kDarkColorScheme = ColorScheme.fromSeed(
+  brightness: Brightness.dark,
+  seedColor: const Color.fromARGB(255, 5, 100, 125),
+);
+
+/// The main ExpenseTrackerApplication widget
+class ExpenseTrackerApp extends StatelessWidget {
+  /// Creates an ExpenseTrackerApplication widget
+  const ExpenseTrackerApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      darkTheme: ThemeData.dark().copyWith(
+        useMaterial3: true,
+        colorScheme: kDarkColorScheme,
+        cardTheme: const CardTheme().copyWith(
+          color: kDarkColorScheme.secondaryContainer,
+          margin: const EdgeInsets.symmetric(horizontal: 16, vertical: 8),
+        ),
+        elevatedButtonTheme: ElevatedButtonThemeData(
+          style: ElevatedButton.styleFrom(
+            backgroundColor: kDarkColorScheme.primaryContainer,
+            foregroundColor: kDarkColorScheme.onPrimaryContainer,
+          ),
+        ),
+      ),
+      theme: ThemeData().copyWith(
+        useMaterial3: true,
+        colorScheme: kColorScheme,
+        appBarTheme: const AppBarTheme().copyWith(
+          backgroundColor: kColorScheme.onPrimaryContainer,
+          foregroundColor: kColorScheme.primaryContainer,
+        ),
+        cardTheme: const CardTheme().copyWith(
+          color: kColorScheme.secondaryContainer,
+          margin: const EdgeInsets.symmetric(horizontal: 16, vertical: 8),
+        ),
+        elevatedButtonTheme: ElevatedButtonThemeData(
+          style: ElevatedButton.styleFrom(
+            backgroundColor: kColorScheme.primaryContainer,
+          ),
+        ),
+        textTheme: ThemeData().textTheme.copyWith(
+              titleLarge: TextStyle(
+                fontWeight: FontWeight.bold,
+                color: kColorScheme.onSecondaryContainer,
+                fontSize: 16,
+              ),
+            ),
+      ),
+      themeMode: ThemeMode.system,
+      home: const ExpensesScreen(),
+    );
+  }
+}
```

Comment:
Ah, moved the style into its own file I see. Nice way to sort it rather than have it in main